README.md

```diff
@@ -105,13 +105,10 @@ Use `docker kill -s $SIGNAL CONTAINER` to send them, where _CONTAINER_ is the co
 For developing and testing APIcast the following tools are needed:
 
 - [OpenResty](http://openresty.org/en/) - a bundle based on NGINX core and including LuaJIT and Lua modules. Follow the [installation instructions](http://openresty.org/en/installation.html) according to your OS.
+   On macOS you can run `brew bundle` to install OpenResty and LuaRocks.
```

Comment:
I'd add this as pre-requisites:
```brew tap openresty/brew
brew tap 3scale/opentracing
brew update```

Reply:
No need. This is already in the Brewfile.

Reply:
Hm, didn't work for me at first :/